src/utils/particle/repulser.ts

```diff
@@ -41,11 +46,12 @@ export class Repulser {
         }
 
         if (container.repulse.clicking) {
-            let repulseRadius = Math.pow(options.interactivity.modes.repulse.distance / 6, 3);
-            let dx = (container.interactivity.mouse.click_pos_x || 0) - particle.position.x;
-            let dy = (container.interactivity.mouse.click_pos_y || 0) - particle.position.y;
-            let d = dx * dx + dy * dy;
-            let force = -repulseRadius / d;
+            const repulseDistance = options.interactivity.modes.repulse.distance;
+            const repulseRadius = Math.pow(repulseDistance / 6, 3);
+            const dx = (container.interactivity.mouse.click_pos_x || 0) - particle.position.x;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['magic numbers' are not allowed: 0](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5011230)